.travis.yml

```diff
@@ -5,9 +5,9 @@ os:
   - osx
 julia:
   - 1.0
-  - 1
+  - 1 # Most recent v > 1.0
 before_script:
- - julia -e 'using Pkg; Pkg.add(PackageSpec(name="Pavito", rev="master"))' # TODO remove when Pavito#master is released.
+ - julia -e 'using Pkg; Pkg.add(PackageSpec(name="Pavito", rev="master"))' # TODO remove when Pavito #master is released.
```

Comment:
You can remove this, Pavito has been release

Reply:
Done!